src/workloads/query/multiplanner/NoSuchField.yml

```diff
@@ -1,9 +1,10 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/query"
 Description: |
-  The goal of this test is to exercise multiplanning. We create as many indexes as possible, and run a
-  query that makes all of them eligible, so we get as many competing plans as possible. Here, we add
-  an additional predicate: {no_such_field: "none"} to guarantee that we hit getTrialPeriodMaxWorks().
+  The goal of this test is to exercise the "max works" case of multiplanning. The test is similar
```

Comment:
Reading this made me wonder: Is this sufficiently distinct from NoGoodPlan.yml? I guess in multi-planning having zero results puts the query into a different category, so maybe it's worth having both even if thet are similar?

Reply:
They do seem very similar and I'm open to removing one.  I don't see zero vs nonzero as a big difference, if they're both using EOF optimization.